### PR TITLE
chore: Suppress alpha warnings in test. Fix entity serialization in test

### PR DESCRIPTION
**Pull request by adchia**
**What this PR does / why we need it**:
The `make test-python` and `make test-python-integration-local` commands were littered with warnings. This suppresses that to make for a better testing experience.

**Note:** test_lambda fails when switching to entity_key_serialization_version=2 so this temporarily keeps it at 1 for that test

### CODE REVIEW

**Review by kevjumba — APPROVED** [review]
/lgtm

File: sdk/python/tests/utils/feature_records.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from pandas.testing import assert_frame_equal as pd_assert_frame_equal
 from pytz import utc
 
-from feast import FeatureStore, utils
+from feast import FeatureService, FeatureStore, utils
```
Comment (achals) [inline sdk/python/tests/utils/feature_records.py:9]: ideally these imports should come from `feast.feature_service`, `feast.feature_store`, etc.

Also,  `utils` :(

File: sdk/python/tests/integration/materialization/test_lambda.py [hunk 2]
```diff
@@ -33,7 +33,10 @@ def test_lambda_materialization_consistency():
         },
         registry_location=RegistryLocation.S3,
     )
-    lambda_environment = construct_test_environment(lambda_config, None)
+    # TODO(adchia): figure out why entity_key_serialization_version 2 breaks with this test
+    lambda_environment = construct_test_environment(
+        lambda_config, None, entity_key_serialization_version=1
+    )
```
Comment (achals) [inline sdk/python/tests/integration/materialization/test_lambda.py:39]: This is probably becase we need to build and push a new docker image (that uses the new version number). The docker image isn't automatically updated via CI at the moment.

File: sdk/python/feast/stream_feature_view.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 import dill
 from typeguard import typechecked
 
-from feast import utils
+from feast import flags_helper, utils
```
Comment (achals) [inline sdk/python/feast/stream_feature_view.py:11]: same import comment.

**Review by achals — APPROVED** [review]

**Pull request merged**